src/views/user/Profile.js

```diff
@@ -1,7 +1,111 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
+import { userUid } from 'src/utils/userUid'
+import { axiosPrivate } from 'src/utils/axiosPrivate'
+import { CCard, CAvatar, CRow, CCol, CCardBody } from '@coreui/react'
+import avatarImg from '../../assets/images/avatars/6.jpg'
 
 const Profile = () => {
-  return <></>
+  const [userDetails, setUserDetails] = useState({ email: '', first_name: '', last_name: '' })
+  // let initials
+  // get user by user_uid
+  const getUser = async () => {
+    const res = await axiosPrivate.get(`/v1/user/${userUid()}/`)
+    setUserDetails(res.data)
+    console.log(res.data)
+  }
+  // initials = userDetails.first_name.substring(0,1).toUpperCase() + userDetails.last_name.substring(0,1).toUpperCase()
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  // format date function
+  const formattedDate = new Date(userDetails.date_joined).toDateString()
+  return (
+    <CRow className="mb-4">
+      <CCol className="col-lg-4">
+        <CCard className="mb-4 shadow-lg">
+          <CCardBody className="card-body text-center">
+            {/* <CAvatar className="mt-4" color="primary" textColor="white" size="xl">
+              { initials }
+            </CAvatar> */}
+            <CAvatar className="rounded-circle img-fluid" src={avatarImg} size="xl" />
+            <h5 className="my-3">{`${userDetails.first_name} ${userDetails.last_name}`}</h5>
+            <p className="text-muted mb-1">{userDetails.email}</p>
+            <p className="text-muted mb-4">User Role</p>
+            <div className="d-flex justify-content-center mb-2">
+              <button type="button" className="btn btn-outline-danger shadow">
+                Delete Profile
+              </button>
```

Comment:
We shall have the delete account functionality in the user settings page. So delete it from here.

Reply:
Okay. Will do.

Reply:
Done